python/cudf/cudf/core/column/datetime.py

```diff
@@ -589,6 +601,18 @@ def as_string_column(
     ) -> "cudf.core.column.StringColumn":
         return self._local_time.as_string_column(dtype, format, **kwargs)
 
+    def __repr__(self):
+        # Arrow prints the UTC timestamps, but we want to print the
+        # local timestamps:
+        arr = self._local_time.to_arrow().cast(
```

Comment:
Maybe a silly question. Why convert to arrow and then attempt to mirror pandas repr conventions instead of just converting it to pandas and using that repr?

Reply:
Mainly because pyarrow has the convenient `to_string()` method that lets us assemble the repr with a custom class name.

```python
In [7]: print(dti._column.to_arrow().to_string())
[
  2001-01-01 05:00:00.000000000,
  2001-01-01 06:00:00.000000000,
  2001-01-01 07:00:00.000000000,
  2001-01-01 08:00:00.000000000,
  2001-01-01 09:00:00.000000000,
  2001-01-01 10:00:00.000000000,
  2001-01-01 11:00:00.000000000,
  2001-01-01 12:00:00.000000000,
  2001-01-01 13:00:00.000000000,
  2001-01-01 14:00:00.000000000
]
```